transformation/src/main/java/org/opensearch/migrations/VersionMatchers.java

```diff
@@ -9,11 +9,15 @@ public class VersionMatchers {
     public static final Predicate<Version> isES_2_X = VersionMatchers.matchesMajorVersion(Version.fromString("ES 2.4"));
     public static final Predicate<Version> isES_5_X = VersionMatchers.matchesMajorVersion(Version.fromString("ES 5.6"));
     public static final Predicate<Version> isES_6_X = VersionMatchers.matchesMajorVersion(Version.fromString("ES 6.8"));
+    public static final Predicate<Version> isES_6_0_to_6_5 = VersionMatchers.equalOrGreaterThanMinorVersion(Version.fromString("ES 6.0"))
```

Comment:
Please follow the existing convention, {operant}{version}{version}

```suggestion
    public static final Predicate<Version> equalOrBetween_ES_6_0_and_6_5 = VersionMatchers.equalOrGreaterThanMinorVersion(Version.fromString("ES 6.0"))
```